bindings/wallet-core/src/wallet.rs

```diff
@@ -188,10 +192,10 @@ impl Wallet {
         self.account.confirm(&id);
     }
 
-    /// get access to all the pending transaction
+    /// Get IDs of all pending transactions produced by `convert`.
```

Comment:
this technically (should at least) return *all* the pending transactions. The comment in the js api is correct though.

Reply:
The initial transactions in block0 are immediately confirmed, where do the others come from?